meson.build

```diff
@@ -638,7 +638,9 @@ endif
 
 ############################################################
 
-if run_command('python3', '-c', 'import jinja2', check : false).returncode() != 0
+pymod = import('python')
+python = pymod.find_installation('python3')
```

Comment:
If you aren't using any of the other functionality of the python module, it's better to use the generic method `find_program('python3')` instead.

(This uses the exact same method for finding the python executable as the python module does, but does less extra work since it doesn't try to figure out how to build and install python libraries.)

Reply:
Updated the PR with this suggestion.